#! /bin/sh /usr/share/dpatch/dpatch-run
## 17_check_smtp_fix_duplicate_cert_message.dpatch
## From 4d06603060fc1233861b164870f0d3a2e0d8d2eb Fri, 15 Jul 2011 20:19:15 +0000 (+0100)
## From: Ton Voon <ton.voon@opsera.com>
## Date: Tue, 6 Sep 2011 23:20:21 -0400
## Subject: [PATCH] Fix check_smtp and check_tcp where duplicate messages were displayed for certificate... 
## X-Git-Url: http://nagiosplug.git.sourceforge.net/git/gitweb.cgi?p=nagiosplug/nagiosplug;a=commitdiff_plain;h=4d06603060fc1233861b164870f0d3a2e0d8d2eb
##
## DP: Fix check_smtp where duplicate messages were displayed for certificate errors

@DPATCH@

--- a/plugins/check_smtp.c
+++ b/plugins/check_smtp.c
@@ -276,9 +276,6 @@ main (int argc, char **argv)
 #  ifdef USE_OPENSSL
 		  if ( check_cert ) {
 		    result = np_net_ssl_check_cert(days_till_exp);
-		    if(result != STATE_OK){
-		      printf ("%s\n", _("CRITICAL - Cannot retrieve server certificate."));
-		    }
 		    my_close();
 		    return result;
 		  }
